### PR TITLE
BACK-1474 get operation by uid endpoint

### PR DESCRIPTION
**Pull request by mbasire-ledger**
## What is this about?

Expose Get operation by uid endpoint

## Cute picture of lama
lama

### CODE REVIEW

File: coins/bitcoin/api/src/it/scala/co/ledger/lama/bitcoin/api/AccountControllerIT.scala [hunk 1]
```diff
@@ -87,199 +96,210 @@ trait AccountControllerIT extends AnyFlatSpecLike with Matchers {
       )
     )
 
+  private def getOperation(accountid: UUID, operationId: Operation.UID) =
+    Request[IO](
+      method = Method.GET,
```
Comment (vbergeron-ledger) [inline coins/bitcoin/api/src/it/scala/co/ledger/lama/bitcoin/api/AccountControllerIT.scala:101]: The default method is GET anyway.

File: coins/bitcoin/api/src/it/scala/co/ledger/lama/bitcoin/api/AccountControllerIT.scala [hunk 2]
```diff
@@ -87,199 +96,210 @@ trait AccountControllerIT extends AnyFlatSpecLike with Matchers {
...
+      uri = Uri.unsafeFromString(s"$serverUrl/accounts/$accountid/operations/${operationId.hex}")
```
Comment (vbergeron-ledger) [inline coins/bitcoin/api/src/it/scala/co/ledger/lama/bitcoin/api/AccountControllerIT.scala:102]:
```suggestion
      uri = Uri.unsafeFromString(serverUrl)/ "accounts"/ accountid / "operations" / operationId.hex
```

Comment (mbasire-ledger) [reply coins/bitcoin/api/src/it/scala/co/ledger/lama/bitcoin/api/AccountControllerIT.scala]: Suggestion integrated. Thanks.

**Review by estrauser-ledger — APPROVED** [review]
Approval given under duress

**Pull request merged**